linalg/simd/x86.hpp

```diff
@@ -12,10 +12,22 @@
 #ifndef MFEM_SIMD_X86_HPP
 #define MFEM_SIMD_X86_HPP
 
+#include "../../config/tconfig.hpp"
+
+#ifndef MFEM_USE_CUDA
```

Comment:
This seems to completely disable intrinsics when cuda is enabled. I think we want to support builds where both cuda and intrinsics are enabled.

If this causes issues with some configurations on some platforms, I'd prefer to disable intrinsics by default but still allow cuda+simd when that works.

Reply:
`ok`